dockertest/subtest.py

```diff
@@ -429,13 +458,62 @@ def try_all_stages(self, name, subsubtest):
                               self.exception_info["error_source"],
                               detail)
             exc_info = self.exception_info["exc_info"]
+            # Ignore known problems in specific docker builds
+            if self.is_known_failure(name):
+                self.final_subsubtests.add(name)
             # Treat N/A as passed, even though this may hide obscure failures
             if isinstance(detail, TestNAError):
                 self.logwarning("Treating TestNAError as PASS")
                 self.final_subsubtests.add(name)
             # cleanup() will still be called before this propagates
             raise exc_info[0], exc_info[1], exc_info[2]
 
+    def is_known_failure(self, subsubtestname):
+        """
+        Do we have a registered known failure in this subtest when running
+        on the currently-installed docker? Return True if so.
+        Side effect: log warning messages to help human debuggers.
+        """
+        # e.g. docker_cli/subtest/subsubtest
+        fullname = os.path.join(self.config_section, subsubtestname)
+        known = known_failures()
+        if fullname not in known:
+            return False
+        docker_nvr = docker_rpm()
```

Comment:
FWIW: Should have been able to dangle it off of ``self`` (the subsubcaller instance) with no prob.  Just in case you do need a wider-scope variable.  That's the place for it.